tap_helpscout/transform.py

```diff
@@ -77,11 +77,46 @@ def transform_conversations(this_json, path=None):
     return this_json
 
 
+def get_rating_value(rating_id):
+    return {"1": "Great", "2": "Okay", "3": "Not Good"}.get(rating_id, "")
+
+
+def transform_ratings(this_json, path):
+    if path is None:
+        return this_json
+
+    for record in this_json[path]:
+        if 'id' in record:
+            record["conversation_id"] = record["id"]
+            record.pop("id")
+
+        # Adds ratings value based on ratingId
+        if "rating_id" in record:
+            record["rating"] = get_rating_value(str(record["rating_id"]))
+
+        record["thread_id"] = record["threadid"]
+        record.pop("threadid")
+
+    return this_json
+
+
+def transform_team_users(this_json, path):
+    for record in this_json[path]:
+        record["user_id"] = record["id"]
```

Comment:
Unless otherwise advised by Product to handle the field in this manner, best-practice is to keep field names consistent with source-service API documentation.

Reply:
Product approved